Everything to get started with mongoDB/README.md

```diff
@@ -0,0 +1,137 @@
+Two more important query operations are `$in` and `$or`.<br>
+The `$in` operator is used for matching one of several values that we pass as an array. For example to find the unicorns that loves apple or orange the command would be `db.unicorns.find({loves : { $in : ['apple', 'orange'] }})`. Run the command and verify the result with the data you inserted.
+> Task: Can you now write a query to get the unicorns with name = 'Radha' or 'Tanmay' or 'Vikash' using `$in` operator?
+<details>
+    <summary>Hint:</summary>
+    db.unicorns.find({name : { $in : ['Radha', 'Tanmay', 'Vikash'] }})<br>
+</details>
+
+The `$or` operator is used if we want to OR rather than AND several conditions on different fields. For example to get all female unicorns who either love apple or name is Rachna the query would be like `db.unicorns.find({gender : 'f', $or [ { loves : 'apple'}, {name : 'Rancha'} ]})`. Observe how the conditions that need to be or'd are put inside an array with the $or operator.
+> Can you now write a query to get the male unicorns who love grapes or weigh greater than or equal to 50kg? 
+<details>
+    <summary>Hint:</summary>
+    db.unicorns.find({gender : 'm', $or : [ {loves : 'grapes'}, {weight : {$gte : 50}} ]})<br>
+</details>
+
+> Note: Did you notice that in the results of all the queries you made there is an _id field in addition to the data you specified? Every document in a collection must have an _id field. You can specify one yourself while inserting the document or let MongoDB generate a value for you which is of type ObjectId. Most of the times you will probably want MongoDB generate it for you. By default the _id field is indexed.
+
+### Activity 7: Removing data ###
+For removing a document from a collection we use `remove()` function. It works similar to find but rather than returning us the matched document it removes it from the collection. Just replace find with remove. Can you now remove the unicorns who are male and weighs greater than or equal to 60kg? Verify if the matched documents are removed from the collection by running `db.unicorns.find({})`({} is the universal selector. It is used to match all the documents). This will return you all the documents in the collection.
+<details>
+    <summary>Hint:</summary>
+    db.unicorns.remove({gender : 'm', weight : {$gte : 60}})<br>
+    Congratulations if you could come up with the correct answer.
+</details>
+
+## Summery ##
```

Comment:
LGTM! Just a typo though. :sweat_smile: It's SUMMARY!

Anyways, I like your content. Let's see what the other reviewers have to day.